tutorials/FIRM.ipynb

```diff
@@ -6,30 +6,29 @@
    "metadata": {},
    "source": [
     "# The FIxed Risk Multicategorical (FIRM) Score\n",
-    "[Taggart, R., Loveday, N. and Griffiths, D., 2022. A scoring framework for tiered warnings and multicategorical forecasts based on fixed risk measures. Quarterly Journal of the Royal Meteorological Society, 148(744), pp.1389-1406.](https://rmets.onlinelibrary.wiley.com/doi/full/10.1002/qj.4266)\n",
+    "The FIRM score is a verification framework for scoring multicategorical forecasts and warnings (Taggart et al. 2022). The framework is tied to pre-defined risk thresholds or probabilistic decision thresholds (for example, issue a yellow warning if the forecast probability for a severe event exceeds 10% and a red warning if the forecast probability exceeds 30%). \n",
```

Comment:
I think this yellow/red warning example aligns with the specific case of FIRM being applied to probabilistic forecasts of binary events, binned into categories (e.g., TS outlooks). This is a very specific case of the FIRM score and not the most common way that we expect it to be used (it doesn't match up to the python example in this notebook) I like the idea of an example, perhaps it could just be reworded to, "forecast the highest warning category that has an x% chance of occurring".  

Potentially we could explore this extension of FIRM at the end of the notebook, but let's consider that another time.

Reply:
No worries, I'll put your suggested text in.